notebooks/114-simplified-mode/114-simplified-mode.ipynb

```diff
@@ -0,0 +1,350 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!mo --framework=onnx --data_type=FP32 --input_shape=[1,3,32,32] -m $onnx_model_path  --output_dir $MODEL_DIR"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Compression stage\n",
+    "Model compression can be performed by calling the following command:\n",
+    "  \n",
+    "`pot -q default -m <path_to_xml> -w <path_to_bin> --engine simplified --data-source <path_to_data>`"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!pot -q default -m $ir_model_xml -w $ir_model_bin --engine simplified --data-source $CALIB_DIR --output-dir compressed --direct-dump "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Compare Performance of the Original and Quantized Models\n",
+    "\n",
+    "Finally, we will measure the inference performance of the FP32 and INT8 models. To do this, we use [Benchmark Tool](https://docs.openvinotoolkit.org/latest/openvino_inference_engine_tools_benchmark_tool_README.html) - OpenVINO's inference performance measurement tool.\n",
+    "\n",
+    "NOTE: For more accurate performance, we recommended running benchmark_app in a terminal/command prompt after closing other applications. Run benchmark_app -m model.xml -d CPU to benchmark async inference on CPU for one minute. Change CPU to GPU to benchmark on GPU. Run benchmark_app --help to see an overview of all command line options."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "optimized_model_path = Path('compressed/optimized')\n",
+    "optimized_model_xml = optimized_model_path / 'resnet20.xml'\n",
+    "optimized_model_bin = optimized_model_path / 'resnet20.bin'"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Inference FP32 model (IR)\n",
+    "!benchmark_app -m $ir_model_xml -d CPU -api async"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Inference INT8 model (IR)\n",
+    "!benchmark_app -m $optimized_model_xml -d CPU -api async"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Demonstration of the results\n",
+    "\n",
+    "In this section the usage of the comressed model will be demonstrated. For this purpose we will run the optimized model on a selected number of pictures from the CIFAR10 dataset and show the predictions of this model.\n",
+    "\n",
+    "In the first step the network is loaded using the IECore:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ie = IECore()\n",
```

Comment:
@Noktyrn, please revise this code using the new OpenVINO API. Here is a guide how to use it: https://docs.openvino.ai/latest/openvino_docs_Integrate_OV_with_your_application.html